share/src/main/java/org/alfresco/web/site/servlet/SSOAuthenticationFilter.java

```diff
@@ -208,10 +208,10 @@ public void init()
         if (endpointDescriptor == null || !endpointDescriptor.getExternalAuth())
         {
             if (logger.isDebugEnabled())
-                logger.debug("No External Auth endpoint configured for " + endpoint);
-            
-            // endpoint is set via bean config - so if no config is using the filter we disable it now
-            this.endpoint = null;
+                logger.debug("No External Auth endpoint configured for " + endpoint);

```

Comment:
As a general rule, please put { } for all 'if' statements to avoid future accidents.  Check sonar recommendations for this.